docs/_content/ontwikkelaars/algemeen/notificaties.md

```diff
@@ -100,3 +127,163 @@ Bronnen:
 * [Digilevering koppelvlakspecificatie](https://www.logius.nl/sites/default/files/public/bestanden/diensten/DigiLevering/Koppelvlakspecificatie.pdf)
 * [Digilevering documentatie](https://www.logius.nl/diensten/digilevering/documentatie)
 * [Regie- en Zaakservices specificaties](https://www.gemmaonline.nl/images/gemmaonline/c/cd/Koppelvlakspecificatie_Regie-zaak_services_v1.0.pdf)
+
+# API specificatie (concept)
+
+[Voorbeeld sequence diagram (in-depth)](http://sequencediagram.org/index.html#initialData=C4S2BsFMAIDkHtQDMQGNICdIDsBQuAHAQw1FRGO2GgC0AlAYUJLIqKrkRBVSNBgbwAtgXjYcwZqTRsOdIgCMFYALIBFKa0rUAIo3z0GAWgB8CZGj4gBw0eKoAuaAAUA8gGUAKtAD0RAiA+AG4AjD4KGGL2ADrYPj4ADADMAEwhRgBeRADWOPG45tyW-NCCImISpvJKqmpODFh8MMDwAaix8clpmTl5PgVcPFY25fbAADxGRtXKwOpOAJI6TiEpSbiGk0aFQyVldhJOABKens7QKQkhHT4AQpHY0ADmmJBPIADOwFiYACbQQkg1CWKzW+VwAF4IQtsGAQERwJ9htAguxoAAqdGGTHQIhIF4PX5Q-B6YxmQbFaylWwVRwuDzePwBYJhRRRSCAqg4G5ddJZXJxfo7SkjA5UKqKWbzaDuACuCg+qAwIAUMFJ0BaGtaaB5qT5vUFAwsvD2NLGWxmtScrgA0rhSVthSaqftacBjqdzpdroKAIIKdmc6hEdgvIQh7LAcFQmFwhFIkqox6Y0k4vEEsREiEGRimJ3I11jJxuLy+fyBULhB72HyrJI+bAU52QD43W6YNAAC0k+dNo0qJktczqLkiv1l6ABLY+RBeWugIOgdaNRWb1P7VAtkqt0FtG0YjqbBbNh2gJzOFwSCRuADV4OAWvAQL98kP1KYTKTiwyy8zK43jWGD4fDEYAkxebA2w7VBu3yIA)
+
+### Abonneren
+
+Een consumer abonneert zich op notificaties door de volgende request naar de
+NC te sturen:
+
+```http
+POST /api/v1/subscribers
+
+Authorization: Bearer abcdef1234
+Content-Type: application/json
+
+{
+    "callbackUrl": "https://ref.tst.vng.cloud/zrc/api/v1/callbacks",
+    "auth": "Bearer aef34gh...",
+    "kanalen": [
+        "zaken",
+        "informatieobjecten"
+    ]
+}
+```
+
+Het `DELETE`n, `PATCH` en `PUT` op individuele subscribers moet mogelijk zijn.
+
+Details moeten ook opgevraagd kunnen worden, met uitzondering van de `auth`
+gegevens.
+
+Filter parameter(s) op `kanelen` zijn handig.
```

Comment:
Als we dan filtering doen, dan moet dit uitgebreid worden:

```suggestion
Filter parameter(s) op `kanalen` zijn handig.

Alle `kenmerken` die een component definieert MOETEN aanwezig zijn in een filter maar de waardes mogen naar wens worden ingevuld zodat er alleen berichten binnen komen die voldoen aan het de filter kenmerken. De waarde `*` betekent dat er niet gefilterd wordt op het element.

Onder de motorkap worden de kenmerken vertaald naar een "topic". Elke waarde wordt base64-encoded en gescheiden door punten. Het voorbeeld wordt dus `MDgyMDk2NzUyMDEx`.`aHR0cHM6Ly9leGFtcGxlLmNvbS9hcGkvdjEvemFha3R5cGVuLzVhYTVj`.`b3BlbmJhYXI=` ofwel als we elk element weer decoden: `082096752011`.`https://example.com/api/v1/zaaktypen/5aa5c`.`openbaar`
```

Reply:
Wat met de `#` dan? Die laat je toe om alles te ontvangen, ongeacht de lengte van het topic.

Reply:
`#` komt eigenlijk overeen met `*.*.*`, als we dat detecteren kan het intern wel `#` worden

Reply:
Niet helemaal - als je `topic` afwijkt van die delen (bijvoorbeeld doordat er 4 zijn ipv 3 oid), dan ontvang je ze wel nog met `#` maar niet met `*.*.*`

Reply:
vandaar dat ik schreef

> Alle `kenmerken` die een component definieert MOETEN aanwezig zijn

We kunnen eventueel zeggen dat helemaal geen filters betekent `#` maar als je ze hebt, moeten alle elementen van het kenmerk er in.

Reply:
prima :+1: